template.yaml

```diff
@@ -220,6 +233,13 @@ Resources:
       Handler: shipLogEntries.shipLogEntries
       Timeout: 5
       Policies:
+      # If this lambda accidentally subscribes to its own log group it can create a feedback loop which overwhelms
+      # Kinesis and spends huge amounts of $$$ on CloudWatch. There is some code which aims to filter out the relevant
+      # log group when creating subscriptions, but we also use this policy to prevent the lambda from sending log events
+      # by default, just to be on the safe side.
+      # If you need to view logs for debugging purposes, the policy below can be temporarily removed from a specific
+      # account using Riff-Raff
```

Comment:
Should we also update the README with similar information? Or at least to say logs for these lambdas cannot be observed by default?

Reply:
Yes, I think a testing/maintenance/debugging section in the README would be helpful. I will add these docs via a separate PR.

Reply:
https://github.com/guardian/cloudwatch-logs-management/pull/113